validation.php

```diff
@@ -0,0 +1,235 @@
+<?php
+
+//=============================================================================
+// FORM VALIDATION
+//contact
+//=============================================================================
+
+
+function validEmail($value)
+{
+	return filter_var($value, FILTER_VALIDATE_EMAIL);	
+}
+
+// =============================================================
+// LOGIN VALIDATION
+//===============================================================
+
```

Comment:
GW: Geef een zelfverklarende naam aan je parameters! $value is bijvoorbeeld ....